src/index.js

```diff
@@ -80,10 +80,25 @@ exports.ensureAuthenticated = (request, response, next) => {
 
 let alreadyInitialized = false;
 
+function buildSchema(Implementation) {
+  const { opts } = Implementation;
+
+  const absModelDirs = opts.modelsDir ? path.resolve('.', opts.modelsDir) : undefined;
+  return requireAllModels(Implementation, absModelDirs)
+    .then((models) => {
+      const integrator = new Integrator(opts, Implementation);
+      return Schemas.perform(Implementation, integrator, models, opts)
+        .thenReturn([models, integrator]);
+    });
+}
+
 exports.init = (Implementation) => {
   const { opts } = Implementation;
+  if (opts.onlyCrudModule === true) {
+    return buildSchema(Implementation).spread(models => models);
```

Comment:
Is it consistent to return `models` for this kind of initialization (with `.spread(models => models);`)?
Do you really need this?

It will return an `app` object in the default initialization that why I am asking.